src/rez/resolved_context.py

```diff
@@ -134,7 +134,7 @@ class ResolvedContext(object):
     command within a configured python namespace, without spawning a child
     shell.
     """
-    serialize_version = (4, 8)
+    serialize_version = (4, 9)
```

Comment:
I'm not too sure if updating the version makes sense now. It should have been updated in the 3.2.0 release.

Reply:
It does make sense since at least a 3.0 would not be able to load with new testing key from this version, even though ideally the last version would have had it.